src/main/java/taxi/controllers/car/GetAllCarController.java

```diff
@@ -0,0 +1,25 @@
+package taxi.controllers.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+public class GetAllCarController extends HttpServlet {
```

Comment:
```suggestion
public class GetAllCarsController extends HttpServlet {
```
that's optional but correct name will be get all cars